src/OrchardCore.Modules/OrchardCore.Workflows/Http/Startup.cs

```diff
@@ -53,6 +53,9 @@ public override void ConfigureServices(IServiceCollection services)
 
             services.AddScoped<ILiquidTemplateEventHandler, SignalLiquidTemplateHandler>();
             services.AddLiquidFilter<SignalUrlFilter>("signal_url");
+
+            services.AddScoped<IWorkflowRoutesProvider, WorkflowTypeRoutesProvider>();
+            services.AddScoped<IWorkflowRoutesProvider, HttpRequestFilterRouteProvider>();
```

Comment:
If `HttpRequestFilter` is part of a specific feature, this call should also be.

Reply:
They are both part of the "OrchardCore.Workflows.Http" feature